tests/cypress/integration/wp-module-support/drawer.cy.js

```diff
@@ -1,7 +1,10 @@
 // <reference types="Cypress" />
+const customCommandTimeout = 30000;
 
 export const CheckDrawerDisabled = () => {
-	cy.get( '.nfd-onboarding-drawer__toggle-button' )
+	cy.get( '.nfd-onboarding-drawer__toggle-button', {
+		timeout: customCommandTimeout,
```

Comment:
Why do we need this to be added? Will it not break the existing TCs?

Reply:
It just increases wait timeout from 5000(cypress default) to 30000. Its not going to overwrite any existing config and break other cases.